packaging/fedora/snapd.spec

```diff
@@ -495,7 +517,8 @@ autoreconf --force --install --verbose
 # selinux support is not yet available, for now just disable apparmor
 # FIXME: add --enable-caps-over-setuid as soon as possible (setuid discouraged!)
 %configure \
-    --disable-apparmor \
+    %{with apparmor}:--enable-apparmor} \
+    %{without apparmor}:--disable-apparmor} \
```

Comment:
`%{?with_apparmor:--enable-apparmor}%{!?with_apparmor:--disable-apparmor}`

Reply:
Thanks, I don't normally do rpm packaging and wasn't sure about this one (it turned out to not work right anyway).